src/OpenTelemetry.Exporter.Geneva/Internal/MsgPackExporter/MessagePackSerializer.cs

```diff
@@ -1,14 +1,16 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#nullable enable
+
 #if NET
 using System.Buffers.Binary;
 #endif
 using System.Globalization;
 using System.Runtime.CompilerServices;
 using System.Text;
 
-namespace OpenTelemetry.Exporter.Geneva;
+namespace OpenTelemetry.Exporter.Geneva.MsgPack;
```

Comment:
nit: I would expect the namespace to match the directory name.
Visual Studio will generate the namespace for new classes based on the directory name. 
I think there should be a high bar for these not matching.

Reply:
I don't necessarily disagree with you, but we're in violation of that all over this repo and the main repo 🤣 For example these files are in an "Internal" folder but we don't have "Internal" in the namespace. Other projects use "Implementation" folders. Kind of a mess. What I did for now is rename the folders here MsgPackExporter -> MsgPack & TldExporter -> Tld that makes them a _little_ less weird. Let's punt on the rest for follow-up cleanup PRs?